src/commands/run.ts

```diff
@@ -173,9 +198,31 @@ export class RunCommand extends ApifyCommand<typeof RunCommand> {
                     warning(`You are running Node.js version ${currentNodeVersion}, which is no longer supported. `
                         + `Please upgrade to Node.js version ${minimumSupportedNodeVersion} or later.`);
                 }
+
                 this.telemetryData.actorNodejsVersion = currentNodeVersion;
                 this.telemetryData.actorLanguage = LANGUAGE.NODEJS;
-                await execWithLog(getNpmCmd(), ['start'], { env, cwd });
+
+                // We allow "module" type directly in node too (it will work for a folder that has an `index.js` file)
```

Comment:
all of this should be properly documented

Reply:
![Code - 2024-04-02 at 20 39 55](https://github.com/apify/apify-cli/assets/17960496/14feef77-09fe-4199-bab5-65caa9fffbf6)